src/functions/pollDelegatedBookings/domain/delegated-bookings-cache-reconciler.ts

```diff
@@ -51,3 +57,24 @@ const appRefsAreEqual = (
   sd1: DelegatedBookingDetail,
   sd2: DelegatedBookingDetail,
 ): boolean => sd1.applicationReference === sd2.applicationReference;
+
+const extractCachedBookingsEligibleForDeletion = (
+  cachedDelegatedBookingsSlots: DelegatedBookingDetail[],
+  activeDelegatedBookingsSlots: DelegatedBookingDetail[],
+  todaysDate: DateTime): DelegatedBookingDetail[] => {
+
+  const activeAppRefs = activeDelegatedBookingsSlots.map(delegatedTestSlot => delegatedTestSlot.applicationReference);
+
+  return cachedDelegatedBookingsSlots.filter((bookingSlot) => {
+
+    if (activeAppRefs.includes(bookingSlot.applicationReference)) return false;
+
+    const unzippedSlot = decompressDelegatedBooking(bookingSlot.bookingDetail);
+    const ageOfBooking = new DateTime(unzippedSlot.testSlot.slotDetail.start).daysDiff(todaysDate);
+
+    if (ageOfBooking > NUMBER_OF_DAYS_TO_RETAIN_CACHED_BOOKINGS) {
```

Comment:
You can just return the expression itself

Reply:
done, thanks